godo.go

```diff
@@ -53,8 +53,14 @@ type Client struct {
 	Keys           KeysService
 	Regions        RegionsService
 	Sizes          SizesService
+
+	// Optional function called after every successful request made to the DO APIs
+	onRequestCompleted RequestCompletionCallback
 }
 
+// Successful request callback function
```

Comment:
Please document exported items in accordance with http://blog.golang.org/godoc-documenting-go-code.